setup.py

```diff
@@ -48,8 +48,10 @@
         "deprecated>=1.0,<2.0",
         "docker>=5.0.3,<7.0.0",
         "python-dateutil>=2.1",
-        "grpcio>=1.43.0,!=1.45.0,<2.0",
-        "grpcio-status>=1.43,!=1.45.0",
+        # Restrict grpcio and grpcio-status.  The 1.49.1 version pulls in a version of protobuf that is not compatible.
+        # More details in https://github.com/flyteorg/flyte/issues/3006
+        "grpcio>=1.43.0,!=1.45.0,<1.49.1,<2.0",
```

Comment:
Is it intended to have `<2.0`? After I install flytekit-1.2.3, I got
```
grpcio                   1.50.0
grpcio-status            1.48.2
```
which doesn't seem to be good.

Reply:
1.2.3 does not contain this section. I'm going to create a patch release shortly.

Reply:
Ah I see. But is `<2.0` intended?

Reply:
Hmm, it is there: https://github.com/flyteorg/flytekit/blob/v1.2.3/setup.py#L51-L52

Reply:
BTW I worked on https://github.com/flyteorg/flytekit/pull/1329